website/screens/components/date-input/specs/DateInputSpecsPage.tsx

```diff
@@ -0,0 +1,521 @@
+              </td>
+              <td>400</td>
+            </tr>
+          </tbody>
+        </DxcTable>
+      </DxcStack>
+      <DxcStack gutter="large">
+        <HeadingLink level={4}>Spacing</HeadingLink>
+        <DxcTable>
+          <thead>
+            <tr>
+              <th>Property</th>
+              <th>Element</th>
+              <th>Core token</th>
+              <th>Value</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>
+                <Code>padding</Code>
+              </td>
+              <td>Calendar container</td>
+              <td>
+                <Code>spacing-16</Code>
+              </td>
+              <td>1rem / 16px</td>
+            </tr>
+            <tr>
+              <td>
+                <Code>padding-left/right</Code>
+              </td>
+              <td>Month container</td>
+              <td>
+                <Code>spacing-8</Code>
+              </td>
+              <td>0.5rem / 8px</td>
+            </tr>
+            <tr>
+              <td>
+                <Code>padding-left/right</Code>
+              </td>
+              <td>Year selector</td>
+              <td>
+                <Code>spacing-16</Code>
+              </td>
+              <td>1rem / 16px</td>
+            </tr>
+            <tr>
+              <td>
+                <Code>padding-top/bottom</Code>
+              </td>
+              <td>Separator</td>
+              <td>
+                <Code>spacing-8</Code>
+              </td>
+              <td>0.5rem / 8px</td>
+            </tr>
+          </tbody>
+        </DxcTable>
+      </DxcStack>
+      <DxcStack gutter="large">
+        <HeadingLink level={4}>Sizing</HeadingLink>
+        <DxcTable>
+          <thead>
+            <tr>
+              <th>Property</th>
+              <th>Element</th>
+              <th>Value</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>
+                <Code>height/width</Code>
+              </td>
+              <td>Selected date circle</td>
+              <td>28/28px</td>
+            </tr>
+            <tr>
+              <td>
+                <Code>height/width</Code>
+              </td>
+              <td>Year selection caret</td>
+              <td>24/24px</td>
+            </tr>
+            <tr>
+              <td>
+                <Code>height/width</Code>
+              </td>
+              <td>Calendar top controls</td>
+              <td>24/24px</td>
+            </tr>
+          </tbody>
+        </DxcTable>
+      </DxcStack>
+      <DxcStack gutter="large">
+        <HeadingLink level={3}>Accessibility</HeadingLink>
+        <HeadingLink level={4}>WCAG 2.2</HeadingLink>
+        <DxcList>
+          <DxcText>
+            Understanding WCAG 2.2 -
+            <DxcLink
+              href="https://www.w3.org/WAI/WCAG22/Understanding/keyboard"
+              text="SC 2.1.1: Keyboard"
+            />
+          </DxcText>
```

Comment:
Add newWindow to the links in accessibility section.

Reply:
Done